FunctionalNetStdV2Tests/FunctionalDemo.cs

```diff
@@ -0,0 +1,256 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace FunctionalNetStdV2Tests {
+    [TestClass]
+    public class FunctionalDemo {
+        //Demonstrates simplied use of sync and async functions 
+        // in the elevated world of results.
+        // Plus, shows uplifting the world of dotnet standard Result naive functions into the results world
+        //Scott Wlashin Functional Programming style -- composition
+        //https://www.youtube.com/watch?v=WhEkBCWpDas&t=1291s
+        //# AIM TO SIMPLIFY 
+        // 1. Method chaining railway track style
+        // 2. asynchronous methods handled using async extensions and minimising the number of 'awaits' needed
+        // 3. uplifting non Result<T> returning functions (both sync and async) to the world of results 
+        //# Design is almost monadic (~95%)
+        //# Use of CSharp 7.2 feature
+        //# --> 'in' keyword on extensions Result parameters, to reduce size of call stack 
+
+        [TestMethod]
```

Comment:
XUnit used in all test projects

Reply:
We reviewed your current version of CSharpFunctionalExtensions and noted many changes since when looked at by our team.  Good work.

Reply:
Sorry, I thought this PR for CSharpFunctionalExtensions